### PR TITLE
Add satellite core class

### PR DESCRIPTION
**Pull request by A-Beck**
First stab at providing a core class. Wanted to get feedback before altering existing code.

@itewk @curt-matthews Do either of you have any feedback about naming, other methods to include, etc?

### CODE REVIEW

File: Automate/RedHatConsulting_Satellite6/Integration/Satellite/StdLib.class/__methods__/satellitecore.rb [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  module StdLib
+    module SatelliteCore
+
+      require 'apipie-bindings'
+      include RedHatConsulting_Utilities::StdLib::Core
+
+      SATELLITE_CONFIG_URI           = 'Integration/Satellite/Configuration/default'
+
+      def initialize( handle = $evm)
+        @handle                  = handle
```
Comment (itewk) [inline Automate/RedHatConsulting_Satellite6/Integration/Satellite/StdLib.class/__methods__/satellitecore.rb]: Seems like a lot of spaces betweeen variable name and =

**Comment by itewk** [comment]
@A-Beck just one small formatting thing. Naming seems good to me. Do you want to merge this in as one PR then start going through and updating existing classes to use the StdLib as a seperate PR?

File: Automate/RedHatConsulting_Satellite6/Integration/Satellite/StdLib.class/__methods__/satellitecore.rb (hunk 1 above)
Comment (A-Beck) [reply Automate/RedHatConsulting_Satellite6/Integration/Satellite/StdLib.class/__methods__/satellitecore.rb]: Good catch - will update

**Comment by A-Beck** [comment]
@itewk I adjusted the whitespace in initialize.

Yeah, I wanted to get the core class merged in first, then tackle updating the methods in another PR

**Pull request merged**